smack-extensions/src/main/java/org/jivesoftware/smackx/jingle/transport/jingle_s5b/JingleS5BTransport.java

```diff
@@ -432,8 +432,9 @@ private void handleCandidateUsed(JingleS5BTransportInfoElement info, JingleEleme
         }
 
         if (theirSelectedCandidate == null) {
-            LOGGER.log(Level.SEVERE, "ILLEGAL CANDIDATE ID!!!");
+            LOGGER.log(Level.SEVERE, "Unknown candidateID.");
```

Comment:
It appears that this error message (and others in this file) could need some more context. A simple start would be possibly to simply add `wrapping`.